PowerEditor/src/Parameters.cpp

```diff
@@ -2848,6 +2891,19 @@ void NppParameters::insertPluginCmd(TiXmlNode *pluginCmdRoot, const PluginCmdSho
 
 void NppParameters::insertScintKey(TiXmlNode *scintKeyRoot, const ScintillaKeyMap & scintKeyMap)
 {
+	TiXmlNode * keyComment;
+	keyComment = scintKeyRoot->InsertEndChild(TiXmlComment());

```

Comment:
Why the `C89` declare-then-assign style?

`TiXmlNode* keyComment = scintKeyRoot->InsertEndChild(TiXmlComment());` might be clearer?

Or, better yet:

`TiXmlNode* const keyComment = scintKeyRoot->InsertEndChild(TiXmlComment());` :)